crates/pathfinder/src/bin/pathfinder/config.rs

```diff
@@ -56,6 +56,16 @@ Examples:
     )]
     rpc_address: SocketAddr,
 
+    #[arg(long = "ws", long_help = "Enable WebSocket transport")]
+    ws: bool,
```

Comment:
Naming: should we rather spell these out? i.e. `websocket` and `websocket.capacity`?

Reply:
A word about config option naming convention:
- down the road in some coming breaking release we're gonna clean up an unify at least the rpc related options into one convention, ie. `domain.[subdomain.]option`
- which means that we'll probably have something like:
```
rpc.address
rpc.max-connections
rpc.cors-domains
```
instead of the current
```
http-rpc
max-rpc-connections
rpc.cors-domains
```
So I suggest going straight for `rpc.websocket[.capacity]` which means that at least the websocket related options will remain backward compatible.

Reply:
I tried to follow geth... It uses `ws`...

Reply:
@shramee , np, I guess at some point we all look at geth looking for conventions :sweat_smile:

Reply:
Done for cap too ;)